code/modules/mob/holder.dm

```diff
@@ -27,6 +27,19 @@ var/list/holder_mob_icon_cache = list()
 	..()
 	START_PROCESSING(SSobj, src)
 
+/obj/item/weapon/holder/throw_at(atom/target, range, speed, thrower)
+	if(held_mob)
+		held_mob.forceMove(loc)
+		var/thrower_mob_size = 1
+		if(ismob(thrower))
+			var/mob/M = thrower
+			thrower_mob_size = M.mob_size
+		var/mob_range = round(range * min(thrower_mob_size / held_mob.mob_size, 1))
+		held_mob.throw_at(target, mob_range, speed, thrower)
+		held_mob = null
+	drop_items()
+	qdel(src)
+
```

Comment:
This should probably be moved to a holder_vr, since this touches a polaris file

Reply:
I pushed this upstream to Polaris https://github.com/PolarisSS13/Polaris/pull/7513
Didn't see a reason why you shouldn't be able to yeet diona nymphs too lol